nft_ingester/src/parsers/bubblegum.rs

```diff
@@ -202,24 +229,32 @@ async fn handle_bubblegum_instruction<'a, 'b, 't>(
             let leaf_event = get_leaf_event(logs)?;
             db.transaction::<_, _, IngesterError>(|txn| {
                 Box::pin(async move {
-                    save_changelog_events(gummy_roll_events, slot, txn).await?;
+                    let seq = save_changelog_events(gummy_roll_events, slot, txn)
```

Comment:
I looked at `parse_logs` and I'm not seeing how it would prevent logs from multiple Bubblegum instructions from all being parsed into the same Vec.  I might be misunderstanding but seems like if somehow there were multiple bubblegum instructions in a single transaction, there could be logs from all of them in the transaction's logs.

Maybe this will become irrelevant if we start parsing candyguard instructions instead of using logs?

Reply:
Ok I did not fully know what to do about this so for now I changed it to return a `Vec` containing all the seq numbers for all the events it encounters, and the Bubblegum  instruction parser gets the `max()` out of the `Vec`, which is assuming that there's only one set of logs per Bubblegum instruction.